app/views/candidates/show.html.erb

```diff
@@ -52,9 +52,6 @@
                   </div>
                   <br><br>
                   <div>
```

Comment:
If you have removed button, also remove <div> & <br> tags.
**When ever we remove some code we should check what other things can be reduced from code which is of no further use.**